packages/core/src/contrib/__tests__/Minkowski.test.ts

```diff
@@ -115,3 +116,65 @@ describe("halfPlaneSDF", () => {
     expect(numOf(result)).toBeCloseTo(1, digitPrecision);
   });
 });
+
+describe("convexPartitions", () => {
+  test("Figure 1 from [HM83]", () => {
```

Comment:
Great! I'll add some more global tests during the weekend in a separate PR. I want to try adding non-convex shapes to `src/contrib/__tests__/Constraints.test.ts`.

Reply:
Sounds good!